src/encrypted-metadata/utils.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Checks if the input matches the given schema.
+ * @param {import('@ucanto/core/schema/type.js').Schema} schema
+ * @param {any} input
+ * @returns {boolean} - True if the input matches the schema, false otherwise.
+ * */
+export const matchesSchema = (schema, input) => {
```

Comment:
another one to consider putting in js ucanto. non-blocking

Reply:
I think this exists already `schema.is()` or something...